definitions/features/installer.rb

```diff
@@ -97,6 +97,13 @@ def run(arguments = '', exec_options = {})
     out
   end
 
+  def run_with_status(arguments = '', exec_options = {})
+    cmd_with_arguments = "LANG=en_US.utf-8 #{installer_command} #{arguments}".strip
```

Comment:
This is an interesting one. Have we always forced the `en_US` locale? In the past it was needed to work around bugs, but we no longer need that. The only thing is that we can't run with `LOCALE=C`. However, with EL8 it defaults to `LOCALE=C.UTF-8` if the requested locale isn't found so that check probably doesn't hit anything anymore. I think EL8 doesn't even ship the `C` locale anymore.